samples/servicecontrol/events-subscription/sample.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Monitoring NServiceBus endpoints with ServiceControl events.
+component: ServiceControlContracts
+reviewed: 2016-03-26
+tags:
+ - ServiceControl
+ - Notifications
+related:
+- servicecontrol/contracts
+- servicecontrol/plugins
+- samples/custom-checks/monitoring3rdparty
+---
+
+## Prerequisistes
+
+1. [Install ServiceControl](/servicecontrol/installation.md).
+1. Using [ServiceControl Management Utility](/servicecontrol/license.md#licensing-servicecontrol-servicecontrol-management-utility) tool, set up ServiceControl to monitor endpoints using MSMQ transport.
```

Comment:
Do we need to mention that this should be running on the local machine? Or that the instance should be named `Particular.ServiceControl`? That should be the default but if you already have SC running on a different transport on your machine, when you set up SC it will name the new instance `Particular.ServiceControl1`.